h/static/styles/partials-v2/_share-widget.scss

```diff
@@ -0,0 +1,154 @@
+// Share widget
+// ---------------
+// The widget that allows social linking and the ability to
+// copy a url for use elsewhere.
+
+
+.js-share-widget {
+  // we are explicitly using visibility so that we can
+  // easily calculate height and width without displaying the element.
+  // display:none causes 0 return value on getBoundingClientRect
```

Comment:
As I understand it, the purpose of this styling is to hide the widget before the JS runs, since in the `ShareWidget` constructor you call `hide()` anyway? If I remove this styling and add the `is-hidden-when-loading` class to the `<div>` with the `.js-share-widget`, everything still works as far as I can see. Am I missing something?